spyder/workers/updates.py

```diff
@@ -267,10 +268,13 @@ def _download_installer(self):
         installer_dir_path = osp.join(
             dir_path, self.latest_release_version)
         os.makedirs(installer_dir_path, exist_ok=True)
```

Comment:
```suggestion
            dir_path, self.latest_release_version
        )
        os.makedirs(installer_dir_path, exist_ok=True)

```

Small stylistic improvement.